--- configure.ac	2019-06-25 03:21:00.809391882 +0000
+++ configure.ac.ng	2019-06-25 03:59:01.529464802 +0000
@@ -15,7 +15,7 @@
 
 dnl Checks for libraries.
 
-AC_CHECK_LIB([espeak], [espeak_Initialize], [], [AC_MSG_ERROR([espeak_Initialize was not found in libespeak])])
+AC_CHECK_LIB([espeak-ng], [espeak_Initialize], [], [AC_MSG_ERROR([espeak_Initialize was not found in libespeak-ng])])
 AC_CHECK_LIB([pipcmrender], [pipcmrender_initialize], [], [AC_MSG_ERROR([pipcmrender_initialize was not found in libpipcmrender])])
 AC_CHECK_LIB([pthread], [pthread_mutex_init], [], [AC_MSG_ERROR([pthread_mutex_init was not found in libpthread])])
 
